src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -418,10 +421,15 @@ def _default_update_upload_progress(upload_metadata: Dict[str, str]) -> bool:
     if not upload_progress_callback:
         upload_progress_callback = _default_update_upload_progress
 
+    # Create job history dir for the manifest file(s)
+    manifest_write_dir = os.path.join(
```

Comment:
Is there a way to disable this feature with deadline config? If not I think it would make sense to add that ability.

Also I think it would make sense to have a print giving the local path the manifest was saved during job submission. I think it would be more convenient to the user than having to remember/check the path to the manifest directory, and then know to call `job get` to see what the name of the manifest would be.

Reply:
Yea great idea, I've added the CLI flag `--save-manifests` which will write to a temporary directory, and I added an info log statement to print where manifests are written if this is specified